pydoctor/test/test_mro.py

```diff
@@ -221,3 +221,22 @@ def z():...
     assert isinstance(klass, model.Class)
     assert klass.subclasses == [dimond.contents['A'], dimond.contents['B']]
     assert list(util.overriding_subclasses(klass, 'z')) == [dimond.contents['A'], dimond.contents['C']]
+
+def test_list_inherited_members() -> None:
```

Comment:
Can you add a docstring to this test explaining what is being tested, per https://jml.io/pages/test-docstrings.html ?